Jolt/Math/Math.h

```diff
@@ -122,6 +122,14 @@ inline uint CountTrailingZeros(uint32 inValue)
 	#endif
 #elif defined(JPH_CPU_E2K)
 		return inValue ? __builtin_ctz(inValue) : 32;
+#elif defined(JPH_CPU_PPC64)
```

Comment:
This code is the same as that for JPH_CPU_E2K so it can just be `#elif defined(JPH_CPU_E2K) || defined(JPH_CPU_PPC)`